qml/Aqt/Cassowary/Stay.qml

```diff
@@ -23,6 +23,6 @@ import Aqt.Cassowary 1.0
 
 Edit {
     id: edit
-    suggested: actualTarget.value
+    suggested: actualTarget ? actualTarget.value : NaN
```

Comment:
Yes, it is here. On the other hand, you should not use NaN in `expr:`... It is easy to fix though now that I think about it, so I'm gonna fix that later too...